charts/datagovuk/templates/find/init-configmap.yaml

```diff
@@ -4,19 +4,5 @@ metadata:
   name: {{ .Release.Name }}-find-init
 data:
   init.sh: |
-    while ! pg_isready -d $DATABASE_URL; do
-      sleep 1;
-    done
-
-    bin/rails db:setup
-    bundle exec rails db:create && bundle exec rails db:schema:load
-
-    # wait until CKAN is ready
-    until [ "$health" = 'OK' ]; do
-        health="$(curl -fsSL "$CKAN_URL/healthcheck")"
-        if [ "$health" != 'OK' ]; then
-            echo "CKAN is unavailable - sleeping"
-            sleep 10
-        fi
-    done
-    echo "CKAN is available..."
+    # remove server.pid upon new deployment
+    rm -rf /srv/app/datagovuk_find/tmp/pids/server.pid
```

Comment:
Optional: probably no need to worry about this right now, but it seems odd that this would be necessary given that we'll always be starting with a fresh filesystem. (Also, at some point we're gonna start enforcing `readOnlyRootFilesystem` so eventually you'll need to disable writing pid files and suchlike and/or ensure anything written goes into /tmp or some other ephemeral volume.)

Reply:
yes, I was thinking about this, will look at stopping it from creating the pid in the future but for now just trying to get it to run in integration